src/Components/test/testassets/ComponentsApp.App/ComponentsApp.App.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
+  <!-- Reference targets and props from Components.Build package -->
+  <Import Project="..\..\..\Build\src\ReferenceFromSource.props" />
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
```

Comment:
Why did you remove `ReferenceBlazorBuildLocally` and replace it with a direct import?

Reply:
Ah. I didn't notice there was a difference.